### PR TITLE
2024 Legal Document Refresh

### LINKED ISSUE #5: On sharing copyrighted content
I am not a lawyer and this is not legal advice.

> Sharing any kind of copyrighted content, or any type of content that violates another person's personal, intellectual property or other rights. This includes any content that would violate the Digital Millennium Copyright Act (DMCA).

I understand and agree with the intention of this, however the wording can be improved.

The issue is "[s]haring any kind of copyrighted content". There are three cases in which this is legal and should be allowed:
- The person sharing it is the copyright holder (which is true for any significant portion of original enough text).
- The person sharing it has been authorized by a copyright holder.
- In that specific case, sharing that content falls under an exception to copyright.

Even in these three cases, one is technically sharing copyrighted content, so is in violation of the literal meaning (not the intention) of the first phrase of this clause and, since the rest is in disjunction, of the clause as a whole.

I believe the clause should be rephrased such that its literal meaning aligns with its intention.

### LINKED ISSUE #9: Misinformation/conspiracy theories and hate speech
Explicitly outline these in the AUP:

- Conspiracy theories and misinformation
- Racism, homophobia, transphobia and other forms of hate speech

### LINKED ISSUE #12: doc: Update About.md with the project's current branding and goals
### What happened?

Self explanatory, the title says it all.

The `About.md` file has not been updated in 2 years now, and the website points to this file.

### PR DESCRIPTION
**Pull request by insertish**
it will definitely happen this year

closes #5
closes #9
closes #12

### CODE REVIEW

**Comment by insertish** [comment]
@DeclanChidlow thanks again for drafting up the AUP descriptions
I've merged the suggestions in but held back a few things for discussion:

- These two sections feel like they're providing very specific examples that are already covered by the definitions above them:

  ```diff
  + This extends to the prohibition of:
  + 
  + - Using slurs or derogatory language targeting protected groups
  + - Promoting discriminatory ideologies or theories
  + - Encouraging violence or harm against protected groups
  + - Denying the existence or rights of protected groups
  + - Deliberately misgendering or deadnaming transgender individuals
  ```

  And also with the following section, a lot of these are their own sections in their own right already:

  ```diff
  + Users are responsible for ensuring their activities comply with all applicable laws and ignorance is not a valid defence for violations. Examples of prohibited illegal activities include, but are not limited to:
  + 
  + - Distribution or facilitation of usage of illegal substances
  + - Fraud and financial crimes
  + - Intellectual property infringement
  + - Cybercrime and unauthorized access to systems
  + - Hate crimes and incitement to violence
  + - Revenge pornography
  + - Child exploitation and abuse
  + - Terrorism and extremist activities
  + - Extortion and blackmail
  ```

- I've also re-added the user bot section since it was missing.
- I've also added a section about gathering PII from users without consent.

**Comment by insertish** [comment]
Now closes #12

**Review by DeclanChidlow — APPROVED** [review]

**Comment by JackDotJS** [comment]
is the AUP officially being renamed to "Community Guidelines"? if so, there are a number of references to "AUP" scattered across these documents, which probably need to be replaced (including the filename itself).

File: Acceptable Usage Policy.md [hunk 1]
```diff
@@ -1,59 +1,149 @@
...
+
+  Attempting to circumvent bans or other moderation actions by creating new accounts or using alternative methods to access the platform is prohibited.
+
+- **Engaging in hacking**
+
+  Any attempt to compromise the security of our platform or its users, including phishing attacks, distribution or creation of malware, or Denial of Service attacks, is strictly forbidden.
+
+- **Attempts to hack or otherwise gain unauthorised access to our service**
+
+  For example, accessing another person's account without permission, or attempting to steal another person's account. This also includes attempts to gain access to our backend networks. _Please contact us ahead of time for permission to test our networks by emailing [security@revolt.chat](mailto:security@revolt.chat)._
```
Comment (JackDotJS) [inline Acceptable Usage Policy.md:108]: isnt this pretty much the same thing as the above, with different wording?

**Comment by insertish** [comment]
> is the AUP officially being renamed to "Community Guidelines"? if so, there are a number of references to "AUP" scattered across these documents, which probably need to be replaced (including the filename itself).

Changed everything that can be changed already, not changing file name and URLs for now, that can happen properly later with redirects and what have you.

File: Acceptable Usage Policy.md (hunk 1 above)
Comment (insertish) [reply Acceptable Usage Policy.md:108]: Oh I suppose it is.

**Comment by JackDotJS** [comment]
there's still at least 2 mentions of the AUP in actual text

File: Acceptable Usage Policy.md [hunk 2]
```diff
@@ -1,59 +1,149 @@
...
+  Local laws: Laws applicable in the user's location of access or residence. <br>
+  National laws: Laws of the country where the user is located or where their actions have effect. <br>
+  International laws: Widely recognized international laws and treaties. <br>
+  Laws of our jurisdictions: As outlined in the "Jurisdiction" section below, we are bound by and enforce laws within specific jurisdictions due to our operational locations and data storage.
+
+  Users are responsible for ensuring their activities comply with all applicable laws and ignorance is not a valid defence for violations.
+
+- **Promoting off-platform content that violates our acceptable usage policy**
+
+  Sharing links or references to external content with intent to promote content that violates our AUP is not allowed.
```
Comment (JackDotJS) [inline Acceptable Usage Policy.md:48]: AUP mention

File: Acceptable Usage Policy.md [hunk 3]
```diff
@@ -1,59 +1,149 @@
...
-- Distributing malware such as viruses, trojans, worms, etc. Any software designed with malicious intent to attack another person or to steal their data is not tolerated.
-- Sharing content that is designed to harass or degrade someone else, or that can directly threaten someone's physical, mental or financial state. This includes death threats, doxxing (sharing personal information without permission), or ignoring someone's privacy.
+## Strikes and Suspensions
 
-To report any of the above activity, contact us at [abuse@revolt.chat](mailto:abuse@revolt.chat).
+Violations of this AUP, or other platform terms, may result in strikes against your account. Strikes are irreversible. Account suspensions can be removed through a successful appeal, but only if you believe that we took the wrong action. There is no point asking to be appealed for breaking the law.
```
Comment (JackDotJS) [inline Acceptable Usage Policy.md:132]: AUP mention

**Pull request merged**